base/pop/getpoploglib/other/findhelp.schema.json

```diff
@@ -0,0 +1,54 @@
+{
+  "$id": "https://github.com/GetPoplog/findhelp.schema.json",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "title": "FindHelp",
+  "type": "object",
+  "properties": {
+    "popversion": {
+      "description": "A string describing the version of Poplog.",
+      "type": "string"
+    },
+    "documentation": {
+      "description": "Matching documentation.",
```

Comment:
Matching what?

Reply:
Yes good point! Fixed in the gist. Sadly gists cannot be made organisational & I couldn't think of a sensible place to include this file in the git archive.